uiowa/uiowa
# Followup updates to transportation migration

- [x] Replace hardcoded links
- [x] Check default image style settings
- [x] Decide on default image style





# How to test


Sync D7 site in DevDesktop (transportation.uiowa.edu on uiowa703).
`git checkout transportation-migrate-followup && git pull && composer install`
`blt ds --site=transportation.uiowa.edu`
`drush @transportation.local en transportation_migrate -y`
`drush @transportation.local uli /admin/config/system/sitenow-migrate`
Complete the configuration form using the help text to configure for DevDesktop
`drush @transportation.local ms` -- shows migration status; two migrations should appear, one for articles and one for article redirects
`drush @transportation.local mim --all` -- runs both migrations
There is an ImageMagick error that can be ignored, as can the link replacement notices (no attempt is currently being made at fixing them, so the notice is informational only)

## Main things to check

Find a page with images (such as https://transportation.uiowa.edu/news/recognizing-service-cambus-drivers-transit-operator-appreciation-day from the current site) and check that the images are added as `large__no_crop`.
Edit an article that was migrated and check that the source org/link fields are empty.
